src/js/control-bar/picture-in-picture-toggle.js

```diff
@@ -29,6 +29,20 @@ class PictureInPictureToggle extends Button {
     this.on(player, ['enterpictureinpicture', 'leavepictureinpicture'], (e) => this.handlePictureInPictureChange(e));
     this.on(player, ['disablepictureinpicturechanged', 'loadedmetadata'], (e) => this.handlePictureInPictureEnabledChange(e));
 
+    this.on(player, ['loadedmetadata', 'audioonlymodechange', 'audiopostermodechange'], () => {
+      const isSourceAudio = player.currentType().substring(0, 5) === 'audio';
```

Comment:
This wouldn't work for audio-only HLS

Reply:
Not sure whether anyone actually uses it, but `audio/mpegurl` is a valid type for HLS. I believe it's valid for HLS even if it contains video.

Reply:
the-more-you-know.gif

Yeah, I should be more clear here: the intent is merely to detect cases where we _know_ the media is audio. For HLS or DASH audio-only streams, they may have a blank video track or no video track. Either way, VHS currently does not have a way to detect whether a stream is audio only.

So, we decided that the best course of action for the time being is to do the simple detection we _can_ do.

Reply:
Thanks @harisha-swaminathan!